h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -103,8 +103,9 @@ public int compare(Integer o1, Integer o2) {
               return 0;
             }
           });
-          String [] names2 = new String[names.length];
-          for(int i = 0; i < names2.length-1; ++i)
+          int len = names.length-1;
+          String [] names2 = new String[len]; // this one decides the length of standardized table length
+          for(int i = 0; i < len; ++i)
```

Comment:
Note that the for loop goes from 0 to names2.length-2.  Hence, the intercept term is not assigned anything and therefore set to NA.  I just make sure we generate names2 to be the exact length without the intercept term.  Hence, nothing is assigned to NA.

Reply:
@wendycwong what happens when the model is trained without an intercept (parameter `_intercept = false`) - is this still going to work or will it cause one coefficient to be skipped?

Reply:
Just wanted to make sure the intercept is included in the total lenght even when it was disabled in model training. Perhaps adding a test would be a good idea?

Reply:
Michal:  If you check the original code, the implementation will always assume that there is an intercept term.  That is why the for loop to assign values to name2 goes from 0 to names2.length-2.   I did not change anything in my implementation except to assign name to of length names2.length-1.  So, if you do train without the intercept term, both the original implementation and my implementation here will exclude one predictor.